helm-charts/splunk-otel-collector/templates/config/_common.tpl

```diff
@@ -45,6 +45,17 @@ zipkin:
 {{- end }}
 {{- end }}
 
+{{/*
+Filter Attributes Function
+*/}}
+{{- define "splunk-otel-collector.filterAttr" -}}
+{{- if .Values.logsCollection.containers.useSplunkIncludeAnnotation }}
```

Comment:
I think the problem is because this helper returns `\nsplunk.com/include` or `\nsplunk.com/exclude`. Please trim the space after `if` and `else`

Reply:
```suggestion
{{- if .Values.logsCollection.containers.useSplunkIncludeAnnotation -}}
```

Reply:
Let me try, thanks for the input!